cli/README.md

```diff
@@ -0,0 +1,216 @@
+## divviup - the Divvi Up command line tool
+
+`divviup` is a command line (CLI) tool for doing all basic operations on both the Divvi Up API and Distributed Aggregation Protocol (DAP) API endpoints. It's only likely to work if the leader aggregator is [Janus](https://github.com/divviup/janus). See `divviup --help` for details on all of the commands.
+
+### Command Line Tutorial
+
+First, set an environment variable for the Divvi Up account and an API token. You can create an API token by logging into the [Divvi Up console](https://app.divviup.org).
```

Comment:
The intent here is to enable an easy Divvi Up demo experience. That requires two aggregators, and the only place we have aggregator pairs is staging. So we should consider how we want to enable prospective Divvi Up users to run this demo:

1. Have them use staging Divvi Up (staging `divviup-api`, existing pair of staging Januses).
1.1. We'd have to amend the instructions here to set `--url https://api.staging.divviup.org` when invoking `divviup`
1.2. It'd be confusing for prospective users to have to juggle staging and production accounts, users, credentials
1.3. Our staging environment isn't as available or robust as production
2. Have them use production Divvi Up
2.1. We'd have to find another aggregator to act as the helper (either deploy a new Janus in the production environment(s) or pair one of the staging aggregators)

I think I lean toward the latter option. @philips do you have thoughts?